src/content/structured/components/date-picker/guidance.mdx

```diff
@@ -0,0 +1,154 @@
+  />
+</DoubleDoDontCaution>
+
+Use a date picker to select a date that is known or desired, such as an appointment.
+
+Use a date picker to select a single date where visualising that date could be useful.
+
+Use a date picker when there is a need to view a date in context. For example, if there is a need to see which day of the week a future event occurs.
+
+## When not to use
+
+Avoid using a date picker for well-known dates, such as dates of birth, because it is often easier to type this into a field. For example, if selecting 30/07/1966, consider using [date input](/components/date-input) as a date picker would require many clicks to arrive at the same date.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={figure2}
+    imageAlt="A date picker being used to select a very distant date"
+    caption="Avoid using a date picker when dealing with distant dates."
+  />
+</DoubleDoDontCaution>
+
+## Sizing
+
+Use the date picker size properties to change its vertical size and achieve compact or spacious layouts.
+
+### Width
+
+Use the resizing options to increase or decrease width to match other elements in a form.
+
+Use the [date input](/components/date-input) resizing options to change size of the input and calendar between minimum and maximum widths. For example, to create a larger date picker, resize width of [date input](/components/date-input).
+
+<DoDontCaution
+  imageSrc={figure3}
+  imageAlt="Two date picker examples at their maximum and minimum width"
+  caption="Change the width of the date input to adjust the date picker width."
+/>
+
+## Content
+
+### Labels
+
+Use [content styling](/styles/content-style/principles) for help with building effective labels for people using date input.
+
+### Helper text
+
+Use helper text to indicate restrictions in your date input such as not allowing weekends.
+
+### Placeholder
+
+Use a placeholder to demonstrate an example of the desired input. Do not add essential information into the placeholder as it will be hidden when a date is entered into the field.
+
+## Interaction
+
+Set `open-at-date` to specify the date in view when the date picker is opened. If no date is set, the date picker will default to showing the current date.
+
+The [date input](/components/date-input) and the date picker update according to any input made. For example, if 10/10/2100 is entered in the [date input](/components/date-input), the date picker will open to display the selection.
+
+## Validation
+
+### Standard
+
+See [date input](/components/date-input#validation) for detailed guidance on how to display errors within the date picker.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={figure4}
+    imageAlt="A date picker with error caused by choosing a date outside of allowed range."
+    caption="Make sure to be clear and instructive when writing error messages. Avoid ambiguity or terms like please when writing error messages."
+    state="good"
```

Comment:
This looks like there's a good state overlapping another good state. Might be worth removing the good state from the original image.
<img width="408" alt="Screenshot 2024-05-08 at 08 33 46" src="https://github.com/mi6/ic-design-system/assets/111882034/77011e5c-b002-43de-93a3-db225a28e0f7">